app/models/service_ansible_playbook.rb

```diff
@@ -3,7 +3,11 @@ class ServiceAnsiblePlaybook < ServiceGeneric
 
   # A chance for taking options from automate script to override options from a service dialog
   def preprocess(action, add_options = {})
-    _log.info("Override with new options: #{add_options}") unless add_options.blank?
+    unless add_options.blank?
+      _log.info("Override with new options:")
+      $log.log_hashes(add_options)
```

Comment:
The prefix used when logging is called for within `log_hashes` would be "log_hashes" and not our method, correct?  That would not be useful.